### ISSUE TITLE
Playwright isn't working with ERB

### ISSUE THREAD

**Issue by mcnaveen** [electron-react-boilerplate/electron-react-boilerplate#3162]
## Summary

Can't launch Playwright Browser Instance

Here is how I'm trying to launch the Browser when the Button is clicked inside React App.

```js
<button onClick={() => fetchAddionalData(url)}>Open</button>

```

```javascript
  async function fetchAddionalData(url: string) {
    try {
    const browser = await playwright.chromium.launch({
        headless: true
    });

    const page = await browser.newPage();
    await page.goto(url);
    await page.waitForTimeout(1000);
    await browser.close();
    } catch (error) {
      console.log(error);
    }
  }

```

And this is the error I'm getting inside Electron App Developer Tools Console.

```
TypeError: Cannot read properties of undefined (reading 'launch')
    at fetchAddionalData (Finder.tsx:69)
    at onClick (Finder.tsx:114)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994)
    at invokeGuardedCallback (react-dom.development.js:4056)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:4070)
    at executeDispatch (react-dom.development.js:8243)
    at processDispatchQueueItemsInOrder (react-dom.development.js:8275)
    at processDispatchQueue (react-dom.development.js:8288)
    at dispatchEventsForPlugins (react-dom.development.js:8299
```

I have tried installing the Playwriting inside `release/app` and also the Root of the Project. But nothing works.

**Comment by amilajack** [comment]
i'll take a look into this. out of curiousity, what are you using playwright for?

**Comment by mcnaveen** [comment]
Hi @amilajack 

Actually, I have tried both Puppeteer and Playright both causing the issue.

I'm trying to scrape the data on headless mode when the user clicks the button inside the react app.

But just now I have confirmed this issue is not directly related to ERB.

Is there any way I can run another Express Server along with the ERB so I can create a separate Express/Puppeteer Endpoint to access it from the React App.
